lua/orgmode/colors/markup_highlighter.lua

```diff
@@ -423,8 +442,10 @@ local function apply(namespace, bufnr, line_index)
 end
 
 local function setup()
-  for _, marker in pairs(markers) do
-    vim.cmd(marker.hl_cmd)
+  for delimiter, marker in pairs(markers) do
+    vim.cmd(string.format(marker.hl_cmd, marker.hl_name))
+    vim.cmd(string.format('syn keyword %s_delimiter %s', marker.hl_name, delimiter))
```

Comment:
I don't think we need this. `syn` is used for old Vim syntax highlighting.

Reply:
You are, in fact, correct. I had a misunderstanding on how highlights work in Vim... for several years now lol. Thanks for showing me that, will resolve ASAP.

Reply:
Done.